test/test_ops.py

```diff
@@ -233,6 +233,22 @@ def test_arange_big(self):
   def test_arange_4096(self):
     helper_test_op([], lambda: torch.arange(4096, dtype=torch.int32), lambda: Tensor.arange(4096), forward_only=True)
 
+  def test_linspace(self):
```

Comment:
How many of these tests actually test unique behaviors? More isn't always better